Engine/Helper.cs

```diff
@@ -92,6 +93,8 @@ public static Helper Instance
         /// </summary>
         private Dictionary<Ast, VariableAnalysis> VariableAnalysisDictionary;
 
+
+        public List<String> AvailableCmdletsOnNano; 
```

Comment:
I just realized that it may be better if we use a hashset instead of a list here.